Tests/Python/PythonPackagesTests.cs

```diff
@@ -0,0 +1,516 @@
+﻿/*
+ * QUANTCONNECT.COM - Democratizing Finance, Empowering Individuals.
+ * Lean Algorithmic Trading Engine v2.0. Copyright 2014 QuantConnect Corporation.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+*/
+
+using NUnit.Framework;
+using Python.Runtime;
+using System;
+
+namespace QuantConnect.Tests.Python
+{
+    [TestFixture, Category("TravisExclude")]
+    public class PythonPackagesTests
```

Comment:
👏 

I love it! I  think where possible, we should favor unit tests over regression tests as they provide better documentation, run faster, and are more focused on the system under test. Great work @AlexCatarino!